packages/addon/public/tbpro.js

```diff
@@ -0,0 +1,13 @@
+const btnSignIn = document.querySelector('#signin');
+
+btnSignIn.addEventListener('click', (event) => {
+  event.preventDefault();
+
+  console.log(`You clicked sign in, and tbpro.js is handling it.`)
+
+  browser.runtime.sendMessage({
+    type: 'SIGN_IN',
+  });
```

Comment:
This simply sends a message to background.js, which will actually open the sign in page in a new tab.